configure.ac

```diff
@@ -158,7 +165,9 @@ dnl Now make the Makefiles
 dnl ######################################################################
 
 AC_CONFIG_FILES([Makefile
-                tests/acceptance/Makefile])
+                package_modules/Makefile
+                tests/acceptance/Makefile
+                tests/unit/Makefile])
```

Comment:
No, it's not a Makefile, it's m4 and shell script. While some shell magic might be possible here, not all of these occurrences are as trivial as this one (check out core for example), so I'd rather keep the standard.